### ISSUE TITLE
Do we have someone that works with (any) Government/regulators or those that work closely with gov/reg on policy engaged here?

### ISSUE THREAD

**Issue by vaughanknight** [Green-Software-Foundation/software_carbon_intensity#30]
Do we have someone that works with (any) Government/regulators or those that work closely with gov/reg on policy engaged here?  I think it would be good to get a perspective from the other side, and at some point understanding what would incentivise and drive adoption (which would then hand over to the community WG) 

I've worked over the last few years with people across these groups that work on driving policy, and with groups driving policy with policy makers and they would give quite a bit of insight.  But I'm not sure if we already had someone.

**Comment by jawache** [comment]
There are a few people we've been talking to and I totally agree. I think we are close to getting to a point with the spec where it might help to bring someone in, I'd prefer to get a little more concrete on the details through one or two worked examples so we have some meat to the discussion and tools to make sure everyone is on the same page.

**Comment by Sealjay** [comment]
I'm with you @vaughanknight  - I've worked with and have contacts, but not actively speaking to anyone right now :)

**Comment by Sealjay** [comment]
We could speak to some of the other groups at the Linux Foundation that work with those groups too perhaps? 
https://github.com/finos/open-regtech-sig - Regulation SIG
And more climate related-
https://www.os-climate.org/open-source-and-the-linux-foundation/ or https://wiki.hyperledger.org/display/CASIG/Climate+Action+and+Accounting+SIG+Home

**Comment by vaughanknight** [comment]
Yep.  I have a few opportunities and may talk to some that I'm across to see if it's of interest, or who they think may find it of interest.

**Comment by atg-abhishek** [comment]
Thank you for bringing this up @vaughanknight and agreed with @jawache that once we have a bit more of a `v0.75+` we can move towards bringing some of those folks in to get a regulatory perspective that will help drive the adoption of this. I'd like to leave this issue open for others who have any useful comments / contacts that they'd like to contribute **async** over here.

**Comment by atg-abhishek** [comment]
And sounds like a plan @seanmcilroy29 - we can definitely cross-collaborate across sister foundations. If there are relevant contacts there, we can capture them on this thread as well. Thanks!
